src/main/scala/castalia/JsonConverter.scala

```diff
@@ -0,0 +1,31 @@
+package castalia
+
+import java.io.FileNotFoundException
+import java.net.URL
+
+import spray.json._
+
+import scala.util.Try
+
+/**
+  * Created by jens on 13-12-15.
+  */
+object JsonConverter {
+  def parseJson[T: JsonReader](filename: String): T = {
+    val resource: URL = getClass.getResource("/" + filename)
+    resource match {
+      case url : URL => unmarshalJsonToClass[T](url) getOrElse
+        (throw new UnmarshalException(s"Type could not be unmarshelled from $url"))
```

Comment:
Do we want to throw `Exceptions` here? Another way could be to make it explicit with a `Try` for example